pcdsdevices/stopper.py

```diff
@@ -0,0 +1,137 @@
+from enum import Enum
+
+from ophyd import Component as Cpt
+from ophyd import Device, EpicsSignal, EpicsSignalRO
+
+from .inout import InOutPositioner, InOutPVStatePositioner
+from .interface import BaseInterface, LightpathMixin
+
+
+class Commands(Enum):
```

Comment:
`IntEnum` plays a bit nicer in general, pretending its values are easily casted as int:
```suggestion
class Commands(IntEnum):
```

Reply:
I'll change this here and in valve.py